.travis.yml

```diff
@@ -10,6 +10,13 @@ cache:
   - node_modules # NPM packages
 before_install:
   - |
+        git config --global user.email "travis@welder.web"
+        git config --global user.name "Travis CI"
+        git remote add welder https://github.com/weldr/welder-web.git
+        git fetch welder
+        git log -1 --pretty=format:%H welder/master
+        git rebase welder/master || git rebase --abort
```

Comment:
I would let this fail hard and abort the test early. On a rebase failure the PR cannot be merged anyway, so there is little point in running the tests.